lib/core/src/lib/about/about.component.ts

```diff
@@ -93,7 +132,7 @@ export class AboutComponent implements OnInit {
 
     setECMInfo() {
         this.discovery.getEcmProductInfo().subscribe((repository) => {
-            this.repository = repository;
+            this.repository = repository as RepositoryInfo;
```

Comment:
It is because in this version of `RepositoryInfo`, the modules property is a lightweight version of the js-api `ModuleInfo` because the about component does not require the full `ModuleInfo` interface.

Reply:
But since the RepositoryInfo coming from the js-api is a superset of this one, I would have expected it to work without any explicit casting 🤔